packages/types/src/request-logic-types.ts

```diff
@@ -287,6 +287,4 @@ export enum ROLE {
   PAYEE = 'payee',
   PAYER = 'payer',
   THIRD_PARTY = 'third-party',
-  PAYEE_DELEGATE = 'payeeDelegate',
-  PAYER_DELEGATE = 'payerDelegate',
```

Comment:
I understand the global change, but I don't get why you're deleting these roles, if you could explain please.

Reply:
yeah sorry. it's just that it's nevery used, so cleaning this up

Reply:
@benjlevesque that's what some builder was considering to use, so do you think it could be added back ?

Reply:
I just removed the enum which was useless. the feature is still active